### PR TITLE
Varia: Remove code that changes the cover block colors

### LINKED ISSUE #5182: Varia Themes: Multuple issues with handling of Cover Block colors
This is an umbrella issue to gather multiple open issues related to the current handling of Cover blocks by Varia. 

1. At some point it appears that the default text color display of Varia themes became different between the editor (where it appears white) and the front end (where it appears dark grey). The symptoms of this are reported in 2 different issues:

o https://github.com/Automattic/themes/issues/5181
o https://github.com/Automattic/themes/issues/4993

2. It appears that Varia themes don't adjust the font color for contrast when the opacity of the cover overlay changes, or the darkness of the background color itself changes.. 

o https://github.com/Automattic/themes/issues/5051

### PR DESCRIPTION
**Pull request by scruffian**
#### Changes proposed in this Pull Request:
Removes the code that deals with the colors in the cover blocks. It can be tested on this site: PdA1qf-f-p2

This seems to solve the initial report, but it will also change the colors of cover blocks on sites where users haven't customized them, so it feels quite risky. Not sure if there's another way...

#### Related issue(s):
Fixes https://github.com/Automattic/themes/issues/5182 and #4993

### CODE REVIEW

**Review by pbking — APPROVED** [review]
Due to all the auto-formatting the change was a bit hard to follow but I think it just comes down to [this](https://github.com/Automattic/themes/pull/5212/files#diff-3168dce0e379e1cc5b0bd22433a9f997f3582dae7b03c7385c411b1710ce22ff) .scss file and [these](https://github.com/Automattic/themes/pull/5212/files#diff-2006c5daa354cd29b030f89bf94866d73c13040de08bd83bf30f2f81a9fb5f8a) PHP changes.

Those LGTM.  Removing those styles is the right call; that's Gutenberg business.

**Comment by MaggieCabrera** [comment]
Yeah, I think this should work like this, I think it's fine if the color annotations don't affect the cover block anyway, let the editor and the user handle it.
